src/components/index.ts

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
can we group these components under  a component folder form or formElements?

Reply:
Yes.